front/src/layouts/footer/footer.html

```diff
@@ -0,0 +1,43 @@
+<div class="body">  
+  <div class="commonMainColorBackground main">  
+    <div class="logo">
+      <logo/>
+      <div class="stores">
+        <a>
+          <img alt="App Store" src="../../assets/appStore.png">
+        </a>
+        <a>
+          <img alt="Google play" src="../../assets/googlePlay.png">
+        </a>       
+      </div>    
+    </div>
+    <span>
+      <el-button class="commonMainColorBackground">Фильмы</el-button>
+      <el-button class="commonMainColorBackground">Сериалы</el-button>
+      <el-button class="commonMainColorBackground">Мультфильмы</el-button>
+      <el-button class="commonMainColorBackground">Новости</el-button>
+      <el-button class="commonMainColorBackground">Актеры</el-button>      
+    </span>
+    <div class="socialNetworks">
+      <div>Социальные сети</div>
+      <a>
+        <img alt="Google" src="../../assets/google.png">
+      </a>
+      <a>
+        <img alt="Twitter" src="../../assets/twitter.png">
+      </a>
+      <a>
+        <img alt="Facebook" src="../../assets/facebook.png">
+      </a>
+    </div>   
+  </div>
+  <div class="lowerPart commonLowerPartFooterBackground">
+    <span class="lowerPart allReles">
+      (с) Все права защищены
+    </span>
+    <el-button class="commonLowerPartFooterBackground">Пользовательское соглашение</el-button>
```

Comment:
class name in kebab case like common-part-footer-backround
this name seems strange, are you sure that it doesn't have some meaning, not structure name